app/main.py

```diff
@@ -41,3 +46,26 @@ def make_change(transaction_id : int, transaction: Transaction):
         return {"Error": "Transaction already exists"}
     transactions[transaction_id] = transaction
     return calculate_change(transactions[transaction_id].currency_inserted, transactions[transaction_id].price,  [25, 10, 5, 1])
+
+@app.put("/update-transaction/{transaction_id}")
+def update_transaction(transaction_id: int, transaction: UpdateTransaction):
+    if transaction_id not in transactions:
+        return {"Error": "Transaction does not exist"}
+
+    if transaction.item_name != None:
```

Comment:
All of this if logic will change when you do that transaction update I was discussing with you above.

Reply:
I removed the brackets and quotes around my item_name, price, and currencty_inserted, and put periods in front of them. I will make a new branch and push my changes up to it once I have everything done.

Reply:
line 52: transactions[transaction_id].item_name = transaction.item_name
line 55: transactions[transaction_id].price = transaction.price
line 58: transactions[transaction_id].currency_inserted = transaction.currency_insterted

Reply:
No need for a new branch, push it to this same one. We haven't merged this and it relates to the same work therefore we use the same branch. It also makes it easy to follow what is going on, new branch for every push will get people confused super fast.